src/components/CreatorModal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  CreatorModalStyling,
+  AboutTheCreatorDiv,
+  DataWrapper,
+  NameDiv,
+  LocationDiv,
+  DescriptionDiv,
+  BottomSectionDiv,
+  WebsiteDiv,
+  CollaboratorsDiv,
+  SingleCollaborator
+} from './styled/CreatorModalStyles.jsx';
+
+
+const CreatorModal = ({ creator, collaborators }) => {
+  const { totalBackers, imageURL, description, name, location, website } = creator;
+  const avatarStyle = { borderRadius: "50%", width: "10%", height: "10%" };
```

Comment:
This is a little bit nitpicky, but the goal of styled components is to keep styling information near the code without having to do inline styling. There's nothing wrong with a little inline styling here and there, but thought I'd just mention it.